blocks/lede/lede.js

```diff
@@ -0,0 +1,13 @@
+/*
+ * Lede Block
+ * The opening paragraph(s) of an article. Its text is larger and stands
+ * out from the default paragraph text.
+ */
+export default function decorate(block) {
+    // Row of content from the block; typically one or more paragraphs.
+    // Converts HTMLCollection to array of nodes so it's usable by replaceChildren.
+    const contentArray = Array.from(block?.children?.[0]?.children?.[0]?.children || []);
+
+    // Replace starting markup within our parent block div, to remove excess div wrappers.
+    block.replaceChildren(...contentArray);
```

Comment:
if we did `block.replaceWith` or `block.parentElement.replaceWith` could we get rid of one more div? 

<img width="243" alt="Screenshot 2025-05-09 at 1 28 49 PM" src="https://github.com/user-attachments/assets/252207f6-428a-4eb9-86ee-52c252d9510e" />